docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -0,0 +1,599 @@
+            "10000/10000 [==============================] - 0s 40us/sample - loss: 1.1841 - accuracy: 0.6557\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "qSyjWQ3mPKT9",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## 4. Visualize the results in TensorBoard's HParams plugin"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "5VBvplwyP8Vy",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The HParams dashboard can now be opened. Start TensorBoard and click on \"HParams\" at the top."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "Xf4KM-U2bbP_",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "%tensorboard --logdir logs/hparam_tuning"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "UTWg9nXnxWWI",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"images/hparams_table.png?raw=1\"/>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "4RPGbR9EWd4o",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The left pane of the dashboard provides filtering capabilities:\n",
+        "- Filter which hyperparameters/metrics are shown in the dashboard\n",
+        "- Filter which hyperparameter/metrics values are shown in the dashboard\n",
+        "- Filter on run status (running, success, ...)\n",
+        "- Sort by hyperparameter/metric in the table view\n",
+        "- Number of session groups to show (useful for performance when there are many experiments)\n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "Z3Q5v28XaCt8",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The HParams dashboard has three different views, with various useful information:\n",
+        "* The **Table View** lists the runs, their hyperparameters, and their metrics.\n",
+        "* The **Parallel Coordinates View** shows each run as a line going through an axis for each hyperparemeter and metric. Click and drag on any axis to highlight only sessions that pass through it. This can be useful for identifying which groups of hyperparameters are most important.\n",
+        "* The **Scatter Plot View** shows plots comparing each hyperparameter/metric with each metric. This can help identify correlations. Click and drag to select a region in a specific plot and highlight those sessions across the other plots. "
+      ]
+    },
+    {
+      "metadata": {
+        "id": "fh3p0DtcdOK1",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Download a set of logs with more experiments:"
```

Comment:
Nice idea of making the tutorial lightweight and providing downloadable summaries.

Reply:
Thanks! It's better than waiting 30 minutes for the tutorial to run :).